module2/homework/transformContainers/function.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include <algorithm>
+#include <deque>
+#include <iterator>
+#include <list>
+#include <map>
+#include <string>
```

Comment:
Do you need all of includes here? Generally you want to have as little as possible includes in hpp in favour of cpp(To reduce change that you will load in cpp library that you are not using).

Reply:
+1

Reply:
I could remove `#include <iterator>` and `#include <algorithm>` although the rest must stay because they pop up in function difinition, right?